zebra/zebra_rib.c

```diff
@@ -551,7 +551,9 @@ void rib_install_kernel(struct route_node *rn, struct route_entry *re,
 	 * know that they've lost
 	 */
 	if (old && (old != re) && (old->type != re->type))
-		zsend_route_notify_owner(old, p, ZAPI_ROUTE_BETTER_ADMIN_WON);
+		zsend_route_notify_owner(old, p, ZAPI_ROUTE_BETTER_ADMIN_WON,
+					 (info) ? info->afi : 0,
```

Comment:
How is it possible for info to be null here?  Especially since we've dereferenced it above.